scripts/asm-installer/asm_vm

```diff
@@ -1260,7 +1260,8 @@ retrieve_expansiongateway() {
 retrieve_cluster_network() {
   info "Retrieving the cluster's network name..."
 
-  NETWORK="$(retry 2 run gcloud container clusters describe "${CLUSTER_NAME}" \
+  # TODO possible to share code with scriptaro? or read from something in-cluster to make this more stable.
```

Comment:
I think we should just read from meshconfig in the cluster to get the value.

Reply:
It doesn't exist in meshconfig, it resides in:
* injection template for "cluster wide" network setting 
* gateway env variables

The eastwest gateway env variable would be the most accurate thing to use imo.

Reply:
That sg. Since we already check the existence of eastwest gateway, so we can get the value from there.